governance/polls/Proposal - Increase System Surplus FLOP Threshold

```diff
@@ -0,0 +1,37 @@
+---
+title: Increase the System Surplus FLOP Threshold to 2 Million Dai - August 10, 2020
+summary: Signal your support for increasing the system surplus FLOP threshold from 500,000 to 2,000,000
+discussion_link: https://forum.makerdao.com/t/3316
+vote_type: Plurality Voting
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Poll: Increase the System Surplus FLOP Threshold to 2 Million Dai - August 10, 2020
+
+The Governance Facilitators have placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of the Maker Governance Community. The community can use this poll to express support or opposition to increasing the system surplus FLOP threshold from 500,000 to 2,000,000.
```

Comment:
Likewise change the name. I'd go into a bit more detail here too. Maybe something along the lines of 'Surplus Auction Buffer, also known as the `hump` parameter, more generally known as 'System Surplus.' Probably try and word that better than I have though.

I just think it would be useful to clarify in this section, potentially link to the technical docs as well.

Reply:
I would also specify 500,000 Dai to 2,000,00 Dai.